src/commands/appSettings/connectionSettings/netherite/getNetheriteLocalProjectConnections.ts

```diff
@@ -0,0 +1,60 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzExtFsExtra, type IActionContext } from "@microsoft/vscode-azext-utils";
+import * as path from "path";
+import { hostFileName, localSettingsFileName } from "../../../../constants";
+import { type IDTSTaskJson, type IHostJsonV2, type INetheriteTaskJson } from "../../../../funcConfig/host";
+import { type ILocalSettingsJson } from "../../../../funcConfig/local.settings";
+import { tryGetVariableSubstitutedKey } from "../getVariableSubstitutedKey";
+
+type NetheriteConnectionKeys = { eventHubsNamespaceConnectionKey?: string, eventHubConnectionKey?: string };
+
+/**
+ * The `host.json` allows you to designate which app settings you want to use for key Netherite resources.
+ * Use to retrieve these values in order to know which app/local settings to set and read.
+ */
+export async function getNetheriteSettingsKeys(context: IActionContext & { projectPath: string }): Promise<NetheriteConnectionKeys | undefined> {
+    const hostJsonPath: string = path.join(context.projectPath, hostFileName);
+    if (!await AzExtFsExtra.pathExists(hostJsonPath)) {
+        return undefined;
+    }
+
+    const hostJson: IHostJsonV2 = await AzExtFsExtra.readJSON(hostJsonPath) as IHostJsonV2;
+    return {
+        // Default: "EventHubsConnection"
```

Comment:
I'm referring to the `value` parameter in the `forEach` callback. Let me check the actual code to give you the exact line number...

Reply:
I don't see the `forEach` callback.  Which file and line number are you referring to?